src/applications/personalization/dashboard/actions/claims.jsx

```diff
@@ -0,0 +1,205 @@
+import * as Sentry from '@sentry/browser';
+import recordEvent from 'platform/monitoring/record-event';
+import { apiRequest } from 'platform/utilities/api';
+import get from 'platform/utilities/data/get';
+
+import {
+  getErrorStatus,
+  USER_FORBIDDEN_ERROR,
+  RECORD_NOT_FOUND_ERROR,
+  VALIDATION_ERROR,
+  BACKEND_SERVICE_ERROR,
+  FETCH_APPEALS_ERROR,
+  FETCH_APPEALS_PENDING,
+  FETCH_CLAIMS_PENDING,
+  FETCH_CLAIMS_SUCCESS,
+  FETCH_CLAIMS_ERROR,
+  UNKNOWN_STATUS,
+} from '../utils/appeals-v2-helpers';
+import { roundToNearest } from '../utils/claims-helpers';
+
+// -------------------- v2 and v1 -------------
+export const FETCH_APPEALS_SUCCESS = 'FETCH_APPEALS_SUCCESS';
+// -------------------- v1 --------------------
+export const FETCH_CLAIMS = 'FETCH_CLAIMS';
+export const FETCH_APPEALS = 'FETCH_APPEALS';
+export const FETCH_STEM_CLAIMS_PENDING = 'FETCH_STEM_CLAIMS_PENDING';
+export const FETCH_STEM_CLAIMS_SUCCESS = 'FETCH_STEM_CLAIMS_SUCCESS';
+export const FETCH_STEM_CLAIMS_ERROR = 'FETCH_STEM_CLAIMS_ERROR';
+export const FILTER_CLAIMS = 'FILTER_CLAIMS';
+export const SORT_CLAIMS = 'SORT_CLAIMS';
+
+export function fetchAppealsSuccess(response) {
+  const appeals = response.data;
+  return {
+    type: FETCH_APPEALS_SUCCESS,
+    appeals,
+  };
+}
+
+export function getAppealsV2() {
+  return dispatch => {
+    dispatch({ type: FETCH_APPEALS_PENDING });
+    return apiRequest('/appeals')
+      .then(appeals => dispatch(fetchAppealsSuccess(appeals)))
+      .catch(error => {
+        const status = getErrorStatus(error);
+        const action = { type: '' };
+        switch (status) {
+          case '403':
+            action.type = USER_FORBIDDEN_ERROR;
+            break;
+          case '404':
+            action.type = RECORD_NOT_FOUND_ERROR;
+            break;
+          case '422':
+            action.type = VALIDATION_ERROR;
+            break;
+          case '502':
+            action.type = BACKEND_SERVICE_ERROR;
+            break;
+          default:
+            action.type = FETCH_APPEALS_ERROR;
+            break;
+        }
+        Sentry.withScope(scope => {
+          scope.setFingerprint(['{{default}}', status]);
+          Sentry.captureException(`vets_appeals_v2_err_get_appeals ${status}`);
```

Comment:
I noticed this is the same sentry call as the claims-status app. Not sure if it would be helpful for the sentry logging to be slightly different to indicate that the error occurred here instead of that app. Maybe adding `dashboard` to the exception message or something like that. Just a thought.

Reply:
I like that idea. I can do this